containers/base/Dockerfile.in

```diff
@@ -34,7 +34,7 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
 # Save GPL source packages
     mkdir -p /srcs && \
     cd /srcs && \
-    apt-get source -d wget git python-zmq ca-certificates pkg-config libpng-dev && \
+    apt-get source --allow-unauthenticated -d wget git python-zmq ca-certificates pkg-config libpng-dev && \
```

Comment:
Done.  It's because Ubuntu can't find the keys to the source git repos.  Since we only download these for licensing reasons and don't actually use them, I think it's fine.  The apt-get installs above are still authenticated.